src/people/widgetViews/OrganizationDetails.tsx

```diff
@@ -136,7 +136,7 @@ const OrganizationDetails = (props: {
     async (user: any) => {
       const pubkey = user.owner_pubkey;
       if (uuid && pubkey) {
-        const userRoles = await await main.getUserRoles(uuid, pubkey);
+        const userRoles = await main.getUserRoles(uuid, pubkey);
```

Comment:
my only thought is that doing this broke staging? @elraphty what do you think

Reply:
Were you able to find out what was?